docs/how-to-configure-new-repository.md

```diff
@@ -16,59 +16,44 @@ for the repository.
-   `foo-approvers` is a child of `foo-triagers`, as it each group always contains 
+   and `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers`, and
+   `foo-approvers` is a child of `foo-triagers`, as it each group always contains
    a subset of people and defines a larger scope of privileges.
 2. Every member of `foo-maintainers` should be included in
-   `foo-approvers` and `foo-triagers` explicitly, with the "Maintainer" GitHub 
-   privileges. This allows repository maintainers to invite new approvers and 
+   `foo-approvers` and `foo-triagers` explicitly, with the "Maintainer" GitHub
+   privileges. This allows repository maintainers to invite new approvers and
    triagers to the team.
 3. The team `foo-triagers` has `Triage` permissions for the repository. If repository
    is using the Project Boards, `foo-triagers` should have `Write` permissions to
    have access to the Project Boards. Note, by not adding members of `foo-triagers` to
    CODEOWNERS file, repository restricts triagers from counting a triagers approvals for
    PRs. See the [Policies](#policies) section that suggest to ensure that `Require
    review from Code Owners` is checked.
-4. The team `foo-approvers` has `Write` permissions for the repository.
-5. The team `foo-maintainers` has `Maintain` permissions for the
-   repository.
-   In individual cases, for example, if this is required to allow maintainers to publish releases independently,
-   the team `foo-maintainers` can be granted `Admin` permissions instead.
-6. Root-level `CODEOWNERS` file on the repository should include superset of
-   people from both `foo-approvers` and `foo-maintainers`.
-
-![image](https://user-images.githubusercontent.com/9950081/57563719-d7b6b300-7355-11e9-9ebb-3c4f549336bc.png)
-
-## Policies
-
-Typically `main` branch is set as default. And protected with the following
-policies:
-
-1. Enable `Require pull request reviews before merging` setting. Make sure
-   `Require review from Code Owners` is checked.
-2. The default setting for `Required approving reviews` is `1` approval. Individual 
-   repositores may opt-in for bigger number of required reviews.
-3. Every repo MUST enforce the CNCF `EasyCLA` check under the
-   `Require status checks to pass before merging` section. (if the check isn't
-   showing, have a CLA manager [enable the check for the
-   repo](https://project.lfcla.com/#/project/a0941000002wBz4AAE/cla)). Check
-   with the OpenTelemetry's [EasyCLA admins](../assets.md#user-content-opentelemetry-managed-assets:~:text=Easy%20CLA) 
-4. It is a good practice to check the `Include administrators` setting.
-5. Repository MUST enforce `Restrict who can push to matching branches` setting
-   to only allow the members of `foo-maintainers` to push to the `main`
-   branch.
-
-![image](https://user-images.githubusercontent.com/9950081/57563714-c5d51000-7355-11e9-80c8-68374e2de2f6.png)
+
+## Repository settings
```

Comment:
I am not sure that adding a new file is beneficial here. The previous section is called "Permissions", yet doesn't mention the actual permissions and does not refer to the new file. And conceptually there's no difference between "how to configure new repo" and "default repo settings" to warrant a 2nd file.

Reply:
the second file was requested by @dyladan https://github.com/open-telemetry/community/pull/1311#issuecomment-1335733993

> I think we should have in this repo the same file but which documents all the default settings a repo is expected to have. That way you can only document the deviations in your repository and link to here for anyone that wants to know the other settings but doesn't have permission to view them.
>
> edit: it would also help with setting up repositories and auditing the settings of existing repositories.

I would defer to you and other @open-telemetry/technical-committee folks for whatever is most convenient since you are setting up the repos.

Reply:
@yurishkuro I removed the new file in a different branch to see what it looks like, is this what you were imagining:

https://github.com/trask/community/blob/admin-rights-inlined/docs/how-to-configure-new-repository.md

Reply:
@trask yep, looks great in one file, thanks!